front/lib/api/assistant/conversation.ts

```diff
@@ -1143,6 +1143,20 @@ export async function* editUserMessage(
           "Unexpected: Message or UserMessage to edit not found in DB"
         );
       }
+      const newerMessage = await Message.findOne({
+        where: {
+          rank: messageRow.rank,
+          conversationId: conversation.id,
+          version: {
+            [Op.gt]: messageRow.version,
+          },
+        },
+      });
+      if (newerMessage) {
+        throw new Error(
```

Comment:
If you throw you won't fix anything you want to emit an error event and return from the function early no?

Throwing would still drop the stream right?